README.md

```diff
@@ -79,6 +80,8 @@ It's planned to have some nifty commands like /assistant which directly communic
 
 Set up a MongoDB instance and note its URL. You can set it to store its data in `database` within this folder (you must first make the folder before starting MongoDB)
 
+Get the trivia lists zip from [here](https://siasky.net/nAHYx0Qe7NFag-RuMZTSGizq5ral6Q6m6BZrSHKzzx7r_g) and extract it so that the triviaLists folder is in the top-level directory.
```

Comment:
The trivia lists haven't been removed from the repo yet tho.

Also, this is permanent right? We could consider putting the lists in their own repo too.

Reply:
My bad, forgot to actually remove the folder before committing. And yes, the download link is permanent. As for a separate repo, yeah it'd be a convenient way to update questions and answers, I guess I can let you go ahead and do that

Reply:
Will look into this after the merge.